src/commands/apps-android-sha-list.ts

```diff
@@ -0,0 +1,59 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+import Table = require("cli-table");
+
+import { Command } from "../command";
+import * as getProjectId from "../getProjectId";
+import { listAppAndroidSha, AppAndroidShaData } from "../management/apps";
+import { requireAuth } from "../requireAuth";
+import * as logger from "../logger";
+
+function logCertificatesList(certificates: AppAndroidShaData[]): void {
+  if (certificates.length === 0) {
+    logger.info(clc.bold("No SHA certificate hashes found."));
+    return;
+  }
+  const tableHead = ["App Id", "SHA Id", "SHA Hash", "SHA Hash Type"];
+  const table = new Table({ head: tableHead, style: { head: ["green"] } });
+  certificates.forEach(({ name, shaHash, certType }) => {
+    const splitted = name.split("/");
+    const appId = splitted[3];
+    const shaId = splitted[5];
+    table.push([appId, shaId, shaHash, certType]);
+  });
+
+  logger.info(table.toString());
+}
+
+function logCertificatesCount(count: number = 0): void {
+  if (count === 0) {
+    return;
+  }
+  logger.info("");
+  logger.info(`${count} SHA hash(es) total.`);
+}
+
+module.exports = new Command("apps:android:sha:list [appId]")
```

Comment:
So I just found out that appId *is* required here.  The special version where we could pass `-` instead and get the SHA for all apps is not yet implemented.